### ISSUE TITLE
Is this project still actively maintained?

### ISSUE THREAD

**Issue by ziliangpeng** [couchbase/forestdb#19]

**Comment by aki-ks** [comment]
I wonder the same thing myself. I am very surprised that although this is Couchbase's primary storage engine, it has not been further developed for years.

**Comment by snej** [comment]
That's incorrect. **ForestDB is not in use by Couchbase.** AFAIK the project is completely dormant.

It was never used in Couchbase Server, and while it was an optional alternative storage engine in Couchbase Lite 1.x, it was not included in 2.0 and later.

**Comment by aki-ks** [comment]
Are you sure? The [documentation](https://docs.couchbase.com/server/current/learn/services-and-indexes/indexes/indexing-and-query-perf.html) claims that ForestDB is the default storage engine for GSI indices.

**Comment by snej** [comment]
I'm sorry, it is actually used in indexes. But it's not used for document storage, and in the Enterprise Edition indexes use a different engine. 

Couchbase Server's primary storage engine is [CouchStore](https://github.com/couchbase/couchstore).
